library/ecp_curves.c

```diff
@@ -4922,7 +4922,7 @@ static inline void carry64(mbedtls_mpi_uint *dst, mbedtls_mpi_uint *carry)
 static int ecp_mod_p192(mbedtls_mpi *N)
 {
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
-    size_t expected_width = 2 * ((192 + biL - 1) / biL);
+    size_t expected_width = BITS_TO_LIMBS(192) * 2;
```

Comment:
This (and all of them) should be 2 * ((192 + biL - 1) /  biL), I think. There are issues requiring round up in 32 bit builds as well.

Reply:
I am confused as to why that is.

 Given that BITS_TO_LIMBS is defined as:

`#define BITS_TO_LIMBS(i)  ((i) / biL + ((i) % biL != 0))`

The statements are equivalent for both 32 and 64 bits.

`2 * ((192 + 64 -1) / 64) ==> 2 * (255/64) ==> 2 * 3 = 6`

And 
`2* ((192 / 64) + 1 if (192%64) is not 0) ==> 2 *(3 + 1 if 0 != 0) ==> 2 * 3 = 6`

Same applies with 32bit lims with the result being 12.

`2 * ((192 + 64 -1) / 64) ==> 2 * (255/64) ==> 2 * 3 = 6`

Reply:
Ok, yep, agreed. This was a comment as I was flying out the door to get home, and whilst driving home I disagreed with it.

Reply:
Yup, it's much clearer to use `BITS_TO_LIMBS()`